### PR TITLE
Install deps using cget instead of cmake-get

### LINKED ISSUE #856: [Testing][Bug]Testing is stuck, boost can't be download
Rocm recipes is outdated, link on boost library is broken, access forbidden
https://dl.bintray.com/boostorg/release/1.72.0/source/boost_1_72_0.tar.bz2

Need to fix dockerfile. Change the way to install boost or to change boost source location.

### PR DESCRIPTION
**Pull request by pfultz2**
This will install boost with the correct URL.

Should resolve #856.

### CODE REVIEW

**Comment by jerryyin** [comment]
This gives me fail to link error in a docker image:

> -- Build files have been written to: /usr/local/cget/build/tmp-fcd9d80f342448c38090d530cfe52fd1/build
Install the project...
-- Install configuration: "Release"
-- Installing: /usr/local/cget/pkg/pfultz2__rocm-recipes/install/etc/cget/recipes//boost
...
-- Installing: /usr/local/cget/pkg/pfultz2__rocm-recipes/install/share//cmake/cget-recipe-utils/cget-recipe-utils-config.cmake
Failed to link: /usr/local/cget/pkg/pfultz2__rocm-recipes/install/share/cmake/cget-recipe-utils/cget-recipe-utils-config.cmake -> /usr/local/share/cmake/cget-recipe-utils/cget-recipe-utils-config.cmake
Failed to build package pfultz2/rocm-recipes
1
CMake Error at install_deps.cmake:7 (message):
  Process failed:
  COMMAND;/usr/local/bin/cget;-p;/usr/local;install;-U;pfultz2/rocm-recipes
Call Stack (most recent call first):
  install_deps.cmake:100 (cget_exec)
  install_deps.cmake:115 (cget)

**Comment by pfultz2** [comment]
> This gives me fail to link error in a docker image

Wheres the dockerfile for this?

**Review by jerryyin — APPROVED** [review]
Giving the approval based on following command succeed:

cmake -P install_deps.cmake --minimum

Abstain from reviewing the actual cmake logic. I'm not all that familiar with cmake logic or history of `install_deps.cmake`.

**Comment by atamazov** [comment]
@shurale-nkn Please review asap, thanks.

File: install_deps.cmake [hunk 1]
```diff
@@ -34,73 +42,79 @@ endif()
...
+    # TODO: Check result
+    message("${PREFIX}/pip install ${ARGN}")
+    cget_exec(
+        COMMAND ${PREFIX}/bin/python ${PREFIX}/bin/pip install ${ARGN}
+    )
+endfunction()
+
+virtualenv_install(cget)
+
 # Set compiler to hcc if not set
```
Comment (atamazov) [inline install_deps.cmake:67]: The comment looks wrong.

**Review by shurale-nkn — APPROVED** [review]
If we are going to drop hcc support LGTM

**Comment by atamazov** [comment]
> If we are going to drop hcc support LGTM

Of course, we no longer support HCC.

**Pull request merged**